plugins/friends.js

```diff
@@ -60,6 +63,28 @@ exports.init = function (sbot) {
         cb(null, graphs[graph] ? graphs[graph].toJSON() : null)
       })
     },
+
+    createFriendStream: function (opts) {
+      opts = opts || {}
+      var start = opts.start || sbot.feed.id
+      var graph = graphs[opts.graph || 'follow']
+      if(!graph)
+        return pull.error(new Error('unknown graph:' + opts.graph))
+      var cancel, ps = pushable(function () {
+        cancel && cancel()
+      })
+      var conf = config.friends || {}
+      cancel = graph.traverse({
+        start: start || sbot.feed.id,
+        hops: opts.hops || conf.hops || 3,
```

Comment:
should `hops` default to 3 here? I think I'd expect it to be 1 if unspecified, because I'd be thinking "friends" would be "immediate friends"